testing/dart/image_filter_test.dart

```diff
@@ -78,12 +132,19 @@ void main() {
     }
   }
 
+  List<ImageFilter> composed(List<ImageFilter> a, List<ImageFilter> b) {
+    return <ImageFilter>[for (final ImageFilter x in a) for (final ImageFilter y in b) ImageFilter.compose(outer: x, inner: y)];
```

Comment:
This list will be a.length * b.length in size? That's pretty big, but I guess constructors and == are fast operations.

Reply:
Style-wise I would split this out as
```
    return <ImageFilter>[
      for (final ImageFilter x in a)
        for (final ImageFilter y in b)
          ImageFilter.compose(outer: x, inner: y),
    ];
```
to make it clearer what is being constructed...

Reply:
When I run the tests locally yeah the image filter tests take an extra second or two to finish as compared to other test files.